tests/codegen/is_val_statically_known.rs

```diff
@@ -0,0 +1,50 @@
+// #[cfg(bootstrap)]
+// ignore-stage1
```

Comment:
huh, is that needed? at stage1 the intrinsic exists and is used by llvm

Reply:
Checked it, and the test passes on my computer. Good catch; I'll remove the ignore. However, the power-of-two codegen test fails on stages 1 *and* 2. I guess Bors doesn't check that. I am getting an unrelated error when trying any codegen test on stage 3.

Reply:
Is there some way I can force `test` to show me the names of the tests it runs, even if they pass? `./x test ui --stage 1 -- statically_known` ignores two tests. The run if I pass `--stage 2` or `--ignored`